Make error on custom kernel 4.2.0-3 (Arch)

Commenting out the line resolves the make error and mba6x_bl still works after suspend/resume.  Is this due to my kernel config? 

I use the pkgbuild from testing repo in ABS and modify only kernel name and using modprobed-db.  Not sure if this info is needed but thought I'd give a heads up on this error just in case.  Love the work you do!

cat /var/lib/dkms/mba6x_bl/39.fc64de1/build/make.log                                                                                                                                                   

DKMS make.log for mba6x_bl-39.fc64de1 for kernel 4.2.0-3-mba (x86_64)
Tue Sep  8 11:52:27 PDT 2015
make -C /lib/modules/4.2.0-3-mba/build M=/var/lib/dkms/mba6x_bl/39.fc64de1/build modules
make[1]: Entering directory '/usr/lib/modules/4.2.0-3-mba/build'
  CC [M]  /var/lib/dkms/mba6x_bl/39.fc64de1/build/mba6x_bl.o
/var/lib/dkms/mba6x_bl/39.fc64de1/build/mba6x_bl.c: In function ‘platform_probe’:
/var/lib/dkms/mba6x_bl/39.fc64de1/build/mba6x_bl.c:336:2: error: implicit declaration of function ‘acpi_video_dmi_promote_vendor’ [-Werror=implicit-function-declaration]
  acpi_video_dmi_promote_vendor();
  ^
cc1: some warnings being treated as errors
scripts/Makefile.build:264: recipe for target '/var/lib/dkms/mba6x_bl/39.fc64de1/build/mba6x_bl.o' failed
make[2]: *** [/var/lib/dkms/mba6x_bl/39.fc64de1/build/mba6x_bl.o] Error 1
Makefile:1386: recipe for target '_module_/var/lib/dkms/mba6x_bl/39.fc64de1/build' failed
make[1]: *** [_module_/var/lib/dkms/mba6x_bl/39.fc64de1/build] Error 2
make[1]: Leaving directory '/usr/lib/modules/4.2.0-3-mba/build'
Makefile:8: recipe for target 'all' failed
make: *** [all] Error 2

--- a/mba6x_bl.c
+++ b/mba6x_bl.c
@@ -333,7 +333,11 @@ static int platform_probe(struct platform_device *dev)
 	backlight_device->props.brightness = INIT_BRIGHTNESS;
 	backlight_update_status(backlight_device);
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 2, 0)
+	acpi_video_set_dmi_backlight_type(acpi_backlight_vendor);
+#else
 	acpi_video_dmi_promote_vendor();
+#endif
 	acpi_video_unregister();
 
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 15, 0)

